config/700-websocket-envoyfilter.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This Envoyfilter is needed to support websocket for DomainMapping.
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: dm-websocket-filter
+  namespace: istio-system
+  labels:
+    app.kubernetes.io/component: net-istio
+    app.kubernetes.io/name: knative-serving
```

Comment:
Should we add the same labels with other configuration like https://github.com/knative-sandbox/net-istio/blob/d17e236d799a12687a4bb29b0d9d1a35592c35d0/config/400-config-istio.yaml#L21-L25 

AFAIK, knative/operator uses `networking.knative.dev/ingress-provider` so we need it at least.